_examples/ai/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: dev
+dev:
+	wrangler dev
+
+.PHONY: build
+build:
+	go run ../../cmd/workers-assets-gen
+	tinygo build -o ./build/app.wasm -target wasm -no-debug ./...
```

Comment:
All other examples are using "big Go" by default now :slightly_smiling_face:

Reply:
about this... apart from the differents in the compilation less space (dont know if big go support more things than tynygo... suppouse not) i was playing with adding in package json a "prebuild": "bash scripts/setup-tinygo.sh", that download the latest version of tinygo (it can be done in cloudfare) and execute later with "$(pwd)/_tinygo/bin/tinygo build -o ./build/app.wasm -target wasm -no-debug ./..." i used "_tinygo" as folder to download to skip this folder from go compilation :)

Reply:
change name... but seems in al the example Makefile use tinygo i'll leave the same... if you dont mind

Reply:
done